cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js

```diff
@@ -1,114 +0,0 @@
-describe('non common configs on location response block and apply undo/redo to changes', () => {
-    beforeEach(() => {
-    cy.createFlow({
-    label: 'Undo/Redo location response configs ',
-    })
- }) 
- it('should configure and test the block', () => {
-    const blockConfigs = {
-        label: 'your location',
-            name: {
-                beforeEdit: 'your_location',
-                afterEditPostFix: '_updated',
-            },
-            resources: {
-                ivr: ' 02_flowers_for_albert.mp3',
-                sms: 'Kindly enter your location',
-                ussd: 'Kindly enter your location',
-            },
-        
-        threshold: 
-        {
-           accuracy_threshold:5,
```

Comment:
no need to retest common configs @pascaleli , we'd just focus on those 02 specific configs for this block